view/theme/frio/templates/nav.tpl

```diff
@@ -38,41 +38,40 @@
 
 				{{* Left section of the NavBar with navigation shortcuts/icons *}}
 				<ul class="nav navbar-nav navbar-left" role="menubar">
-					<li id="nav-communication" class="nav-segment" role="presentation">
+					<li id="nav-communication" class="nav-segment">
 						{{if $nav.network}}
-						<a accesskey="n" role="menuitem" class="nav-menu {{$sel.network}}" href="{{$nav.network.0}}" data-toggle="tooltip" aria-label="{{$nav.network.3}}" title="{{$nav.network.3}}"><i class="fa fa-lg fa-th" aria-hidden="true"></i><span id="net-update" class="nav-network-badge badge nav-notification"></span></a>
```

Comment:
Why the removal of these attributes? I believe they are recommended by Bootstrap.

Reply:
You mean this `role="menuitem"`? Well, I was told so. See also this article: https://marcozehe.de/2019/05/30/wai-aria-menus-use-with-care/